packages/hub/config/structure.sql

```diff
@@ -1036,6 +1050,14 @@ ALTER TABLE ONLY public.prepaid_card_patterns
     ADD CONSTRAINT prepaid_card_patterns_pkey PRIMARY KEY (id);
 
 
+--
+-- Name: push_notification_registrations push_notification_registrations_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.push_notification_registrations
+    ADD CONSTRAINT push_notification_registrations_pkey PRIMARY KEY (owner_address);
```

Comment:
It is possible for an EOA to be connected on more than one device running Card Wallet, which will each register to receive push notifications (e.g. an iPad and an iPhone). Therefore owner address is not a suitable primary key. An alternative could be a compound primary key (owner address + push client id) or a surrogate key.

Reply:
Ok, thanks for letting me know about this case! I made the adjustments and chose an uuid for the primary key.